packages/firebase_in_app_messaging/pubspec.yaml

```
@@ -1,6 +1,6 @@
 name: firebase_in_app_messaging
 description: Flutter plugin for Firebase In-App Messaging.
-version: 0.4.0+1
+version: 1.0.0
```

```suggestion
version: 0.4.0+1
```

This is done at release step